glide-core/src/client/value_conversion.rs

```diff
@@ -42,14 +42,12 @@ pub(crate) fn convert_to_expected_type(
             _ => Err((
                 ErrorKind::TypeError,
                 "Response couldn't be converted to map",
-                format!("(response was {:?})", value),
             )
                 .into()),
         },
         ExpectedReturnType::MapOfStringToDouble => match value {
             Value::Nil => Ok(value),
             Value::Map(map) => {
-                let map_clone = map.clone();
```

Comment:
Glad we're finally getting rid of these nasty map clones. I've been trying to do this for a while without disturbing the existing error messages.